test/src/test/java/jenkins/model/AssetManagerTest.java

```diff
@@ -46,11 +46,4 @@ public void emptyAssetDoesNotThrowError() throws Exception {
         HttpURLConnection httpCon = (HttpURLConnection) url.openConnection();
         assertEquals(HttpURLConnection.HTTP_NOT_FOUND, httpCon.getResponseCode());
     }
-
-    @Test
-    public void handlebarsLoad() throws Exception {
-        URL url = new URL(j.getURL() + "assets/handlebars/jsmodules/handlebars3.js");
```

Comment:
Is there any change in the way the plugins consume handlebars3 now? Do plugin developers need to do any change? or are the plugins going to stop working because of that or trigger a 404 in the browser console?

Reply:
They would have to declare the dependency to Handlebars plugin or use the webpack process that was done on Jenkins War and few plugins.